option-parser.js

```diff
@@ -140,6 +140,7 @@ const config = {
 		// Optionally force all LESS/CSS to be handled modularly, instead of solely having
 		// the *.module.css and *.module.less files be processed in a modular context.
 		config.forceCSSModules = computed('forceCSSModules', enact, config.theme);
+		config.webpackExtraResolvePaths = computed('webpackExtraResolvePaths', enact, config.theme);
```

Comment:
Nice.
Could you change the name "webpackExtraResolvePaths" -> "additionalModulePaths"(CRA style naming)?
Then we will review this PR

Reply:
sure, i'll try to do that in conjunction with the change on the related PR, in the next day or two. thank you for the feedback!